### PR TITLE
updates scripts and xadj_seismogram

### PR DESCRIPTION
**Pull request by danielpeter**

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
The files changed require a buildbot test. Buildbot has started.

**Comment by buildbot-princeton** [comment]
Your changes passed the buildbot test. See [build 421](http://specfem2d-buildbot.princeton.edu/builders/specfem2d_builder/builds/421).

**Comment by danielpeter** [comment]
accept

**Comment by komatits** [comment]
accept

On 22/06/2016 16:03, buildbot-princeton wrote:
> Your changes passed the buildbot test. See build 421
> <http://specfem2d-buildbot.princeton.edu/builders/specfem2d_builder/builds/421>.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/geodynamics/specfem2d/pull/587#issuecomment-227753346>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFjDKc7p4yc3cR7INKkpoewJ1xiAqSBlks5qOUCpgaJpZM4I7zka>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

File: utils/compare_two_images.py [hunk 1]
```diff
@@ -55,18 +55,26 @@
...
 
 try:
-    import skimage as ski
-    version = ski.__version__
+    import skimage
+    version = skimage.__version__
     print("skimage version: ",version)
     v = version.split('.')
+    # determines skimage version function
+    new_version = 0
```
Comment (QuLogic) [inline utils/compare_two_images.py]: You should use `True` and `False` in the future.

Comment (danielpeter) [inline utils/compare_two_images.py]: well, what if they decide in version 14 that a third name might be better... then it's not black or white anymore

Comment (QuLogic) [inline utils/compare_two_images.py]: Then `new_version` would not be the right name for the variable. :wink:

**Pull request merged**